Archives/VolFile.h

```diff
@@ -57,7 +57,8 @@ namespace Archives
 		enum class VolPadding : uint32_t
 		{
 			TwoByte = 0,
-			FourByte = 1 // Default value
+			FourByte = 1,
+			default = FourByte
```

Comment:
Err, `default` is a reserved word. This might cause problems. The syntax highlighting on GitHub kind of points that out here.